cmd/kogito/command/shared/install_operator.go

```diff
@@ -74,7 +93,11 @@ func InstallOperatorIfNotExists(namespace string, operatorImage string, client *
 		return nil
 	}
 
-	//TODO: check if we have operator available at the hub, warn the user and exit
+	if available, err := isOperatorAvailableInOperatorHub(client); err != nil {
+		log.Warnf("Couldn't find if the Kogito Operator is available in the cluster: %s ", err)
+	} else if available {
```

Comment:
why return a error in case of success?

Reply:
This could be an security error like "user doesn't have permission to fetch this resource". It's valueable info to the end user to understand why we failed to fetch for the operator hub.

Then we continue to install the operator using the standard way.